### PR TITLE
AppInsights to work on any environment

### PR DESCRIPTION
**Pull request by dolphincrs**

### CODE REVIEW

File: Childrens-Social-Care-CPD/Program.cs [hunk 1]
```diff
@@ -32,6 +34,10 @@
 var options = new ApplicationInsightsServiceOptions { ConnectionString = "InstrumentationKey=13cbca44-e50d-4444-ab56-6c151cad789d;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/" };
 builder.Services.AddApplicationInsightsTelemetry(options: options);
 
+var telemetryConfiguration = TelemetryConfiguration.CreateDefault();
+telemetryConfiguration.ConnectionString = Environment.GetEnvironmentVariable(SiteConstants.CPD_INSTRUMENTATION_KEY);
```
Comment (nnagepat) [inline Childrens-Social-Care-CPD/Program.cs:38]: Please include changed Constants file in your PR

File: Childrens-Social-Care-CPD/Program.cs [hunk 2]
```diff
@@ -32,6 +34,10 @@
 var options = new ApplicationInsightsServiceOptions { ConnectionString = "InstrumentationKey=13cbca44-e50d-4444-ab56-6c151cad789d;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/" };
 builder.Services.AddApplicationInsightsTelemetry(options: options);
 
+var telemetryConfiguration = TelemetryConfiguration.CreateDefault();
```
Comment (nnagepat) [inline Childrens-Social-Care-CPD/Program.cs:37]: You are setting up telemetryConfiguration, however not using it so it wont work,  we can use something like this

var options = new ApplicationInsightsServiceOptions { ConnectionString = Environment.GetEnvironmentVariable(SiteConstants.CPD_INSTRUMENTATION_KEY) };
builder.Services.AddApplicationInsightsTelemetry(options: options);

Comment (nnagepat) [reply Childrens-Social-Care-CPD/Program.cs:37]: Also, AL needs to pass connection string in CPD_INSTRUMENTATION_KEY  env variable

Comment (albal) [reply Childrens-Social-Care-CPD/Program.cs:37]: CPD_INSTRUMENTATION_KEY is already being passed in the environment.

**Pull request closed without merging**